src/Controls/tests/Core.UnitTests/GridMinMaxIntegrationTests.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Xunit;
+
+namespace Microsoft.Maui.Controls.Core.UnitTests
+{
+    public class GridMinMaxIntegrationTests : BaseTestFixture
```

Comment:
Can you make the src/Controls/tests/Core.UnitTests/Layouts/GridLayoutTests.cs partial and then make this be the other partial.

Also, the new file should be called GridLayoutTests.MinMax.cs.